tests/main/preseed-core20/task.yaml

```diff
@@ -142,9 +142,9 @@ execute: |
   MATCH "^snap/core20/current" < files.log
   MATCH "^snap/snapd/current" < files.log
 
-  MATCH "^var/cache/apparmor/11143a61.0/snap.pc.hook.configure" < files.log
-  MATCH "^var/cache/apparmor/11143a61.0/snap-update-ns.pc" < files.log
-  MATCH "^var/cache/apparmor/11143a61.0/snap-confine.snapd.x1" < files.log
+  MATCH "^var/cache/apparmor/08ba9d86.0/snap.pc.hook.configure" < files.log
+  MATCH "^var/cache/apparmor/08ba9d86.0/snap-update-ns.pc" < files.log
+  MATCH "^var/cache/apparmor/08ba9d86.0/snap-confine.snapd.x1" < files.log
```

Comment:
This dir path is the hash of the `aa_features_id(features);` which is the content of /sys/kernel/security/apparmor/features AIUI. Looking at the test we do not mock the apparmor features dir so changes like this can happen if the underlying kernel changes. If we would create a snapshot of an existing `/sys/kernel/security/apparmor/features/` dir and then run the pressed command in the test with `--apparmor-features-dir=...` it should be more stable [1]

[1] https://forum.snapcraft.io/t/uc20-preseeding/30238

Reply:
@mvo5 thanks for the suggestion, I already did the change.